tests/smoke/helpers_test.go

```diff
@@ -0,0 +1,110 @@
+// +build smoke
+
+package smoke
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"get.porter.sh/porter/pkg/claims"
+	"github.com/carolynvs/magex/shx"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	myBunsRef = "localhost:5000/mybuns:v0.1.1"
+	myDbRef   = "localhost:5000/mydb:v0.1.0 "
+)
+
+func (t Test) PrepareTestBundle() {
+	// This variable isn't set on windows and the mybuns bundle relies on it
+	os.Setenv("USER", "porterci")
+
+	// Check if another test has already set up the test bundle
+	err := shx.RunE("docker", "pull", "localhost:5000/mybuns-installer:v0.1.1")
+	if err == nil {
+		return
+	}
+
+	// Build and publish an interesting test bundle and its dependency
+	t.MakeTestBundle("mydb")
+	t.MakeTestBundle("mybuns")
+}
+
+func (t Test) MakeTestBundle(name string) {
+	err := shx.Copy(filepath.Join(t.RepoRoot, "tests/testdata", name), t.TestDir, shx.CopyRecursive)
+	require.NoError(t.T, err)
+
+	pwd, _ := os.Getwd()
+	defer os.Chdir(pwd)
+	os.Chdir(filepath.Join(t.TestDir, name))
+
+	t.RequirePorter("build")
+	t.RequirePorter("publish", "--reference", myBunsRef)
+}
+
+func (t Test) ShowInstallation(namespace string, name string) (claims.Installation, error) {
+	output, err := t.RunPorter("show", name, "--namespace", namespace, "--output=json")
+	if err != nil {
+		return claims.Installation{}, err
+	}
+
+	var installation claims.Installation
+	require.NoError(t.T, json.Unmarshal([]byte(output), &installation))
+	return installation, nil
+}
+
+func (t Test) AssertInstallationExists(namespace string, name string) claims.Installation {
```

Comment:
minor nit on helper functions that start w/ `Assert...` but use `require` under the hood... thoughts on updating the helper functions to `Require...`?  (Or, at least the ones that do use require.)